test/config.tests.js

```diff
@@ -370,6 +370,18 @@ describe('get config', () => {
     );
   });
 
+  it('should not allow "none" for idTokenSigningAlg', () => {
```

Comment:
the test above is for the alg received from the .well-known and this one is for the configuration/options passed by the user. Is that correct? If so, looks good, and thanks for updating these.

Reply:
yep - I thought I'd add one for the configuration as well